client/packages/common/src/ui/forms/JsonForms/JsonForm.tsx

```diff
@@ -94,7 +94,7 @@ const FormComponent = ({
     user,
   };
 
-  return (
+  return !data ? null : (
```

Comment:
I was finding is misleading - the data was null and I was getting a spinner shown unceasingly